tests/gateway/test_gateway_config_parsing.py

```diff
@@ -65,17 +64,14 @@ def test_api_key_parsing():
 
     assert _resolve_api_key_from_input(string_key) == string_key
 
-    path_for_file = "~/mlflow/gateway/mykey.conf"
```

Comment:
We should not create a file like this. If this test fail before `unlink` is called, that would remove this file unremoved and can cause other tests to fail.